lib/ansible/modules/network/avi/avi_serviceengine.py

```diff
@@ -29,33 +29,39 @@
             - The state that should be applied on the entity.
         default: present
         choices: ["absent", "present"]
+        type: str
     avi_api_update_method:
         description:
             - Default method for object update is HTTP PUT.
             - Setting to patch will override that behavior to use HTTP PATCH.
         version_added: "2.5"
         default: put
         choices: ["put", "patch"]
+        type: str
     avi_api_patch_op:
         description:
             - Patch operation to use when using avi_api_update_method as patch.
         version_added: "2.5"
         choices: ["add", "replace", "delete"]
+        type: str
     availability_zone:
         description:
             - Availability_zone of serviceengine.
+        type: str
     cloud_ref:
         description:
             - It is a reference to an object of type cloud.
+        type: str
     container_mode:
         description:
             - Boolean flag to set container_mode.
             - Default value when not specified in API or module is interpreted by Avi Controller as False.
         type: bool
     container_type:
         description:
-            - Enum options - container_type_bridge, container_type_host, container_type_host_dpdk.
+            - Enum options - CONTAINER_TYPE_BRIDGE, CONTAINER_TYPE_HOST, CONTAINER_TYPE_HOST_DPDK.
```

Comment:
Do you have some docs links that this is really the case? Or (better) also move this into another PR.